org_chart.js

```diff
@@ -63,11 +36,20 @@ if (!tim.skills) {
 }
 tim.skills.push("Protocols")
 
-console.log(`${alan.name} is a ${alan.position} working on ${alan.project}.`)
-console.log(`He can be reached at ${alan.email}`)
-console.log(`${grace.name} is a ${grace.position} working on ${grace.project}.`)
-console.log(`She can be reached at ${grace.email}`)
-console.log(`${donald.name} is a ${donald.position} working on ${donald.project}.`)
-console.log(`He can be reached at ${donald.email}`)
-console.log(`${tim.name} is a ${tim.position} working on ${tim.project}.`)
-console.log(`He can be reached at ${tim.email}`)
+//assign Person
+let elon = new Person('Elon Musk', "elon.musk@musky.com", "Boss", "He")
+let alan = new Person('Alan Turing', "alan.turing@musky.com", "Computer Scientist", "Cryptography", "He");
+let grace = new Person('Grace Hopper', "grace.hopper@musky.com", "Computer Scientist", "Compilers", "She");
+let donald = new Person('Donald Knuth', "donald.knuth@musky.com", "Mathematician", "Algorithm Analysis", "He");
+let tim = new Person("Tim Berners-Lee", "tim.berners_lee@musky.com", "Computer Science", "Networks");
+
+//greeting
+elon.greeting()
+alan.greeting()
+grace.greeting()
+donald.greeting()
+
+//addManager
+alan.addManager(elon)
+grace.addManager(elon)
+donald.addManager(elon)
```

Comment:
Where did the skills get added?
THen you should notice that we are calling `addManager`, `greeting` and `addSkill` for each person.  That generates duplication, so I would either make another function or use some sort of loop to amend that.